go/teams/storage.go

```diff
@@ -71,19 +71,78 @@ func (s *Storage) Get(mctx libkb.MetaContext, teamID keybase1.TeamID, public boo
 	return ret
 }
 
+// Store TeamData's on memory and disk. Threadsafe.
+type FTLStorage struct {
+	*storageGeneric
+}
+
+// Increment to invalidate the disk cache.
+const ftlDiskStorageVersion = 10
+const ftlMemCacheLRUSize = 200
+
+type ftlDiskStorageItem struct {
+	Version int                    `codec:"V"`
+	State   *keybase1.FastTeamData `codec:"S"`
+}
+
+var _ teamDataGeneric = (*keybase1.FastTeamData)(nil)
+var _ diskItemGeneric = (*ftlDiskStorageItem)(nil)
+
+func (d *ftlDiskStorageItem) version() int {
+	return d.Version
+}
+func (d *ftlDiskStorageItem) value() teamDataGeneric {
+	return d.State
+}
+func (d *ftlDiskStorageItem) setVersion(i int) {
+	d.Version = i
+}
+func (d *ftlDiskStorageItem) setValue(v teamDataGeneric) error {
+	typed, ok := v.(*keybase1.FastTeamData)
+	if !ok {
+		return fmt.Errorf("teams.FTLStorage#Put: Bad object for setValue; got type %T", v)
+	}
+	d.State = typed
+	return nil
+}
+
+func NewFTLStorage(g *libkb.GlobalContext) *FTLStorage {
+	s := newStorageGeneric(g, ftlMemCacheLRUSize, ftlDiskStorageVersion, libkb.DBFTLStorage, libkb.EncryptionReasonTeamsLocalStorage, "ftl", func() diskItemGeneric { return &ftlDiskStorageItem{} })
```

Comment:
using the same derived key (reason). ok?

Reply:
yeah i thought it was OK, don't you? It's basically the same data.